filament/backend/src/opengl/GLUtils.h

```diff
@@ -472,6 +472,34 @@ constexpr /* inline */ GLenum getInternalFormat(TextureFormat format) noexcept {
             return 0;
 #endif
 
+#if defined(GL_EXT_texture_compression_rgtc)
+        case TextureFormat::RED_RGTC1:            return GL_COMPRESSED_RED_RGTC1_EXT;
```

Comment:
Small nitpick: the returns are misaligned here

Reply:
f18d72e34936c4f40df4d0ed09dd818017822c5e